app/CustomRoles/roles.php

```diff
@@ -0,0 +1,33 @@
+<?php
+add_action('admin_init', function() {
+if (!get_role('promos_editor')) {
+    // Add Promos editor role
+    add_role('promos_editor',
+      'Promos Editor',
+      array(
+        'read' => true,
+        'edit_posts' => true,
+        'delete_posts' => true,
+        'publish_posts' => true,
+        'upload_files' => true,
+      )
+    );
+  }
+  // Add the roles you'd like to administer the custom post types
+  $roles = array('promos_editor','administrator');
+  // Loop through each role and assign capabilities
+  foreach($roles as $the_role) {
```

Comment:
Normally would rewrite this as `foreach($things as $thing) {` rather than `foreach($things as $the_thing) {`